MoppLib/MoppLib.xcodeproj/project.pbxproj

```diff
@@ -582,7 +585,6 @@
 				C54EA736204D90D50039AC78 /* MoppLibCardReaderManager.h in Headers */,
 				545CF8571E16928600B66EE8 /* CBManagerHelper.h in Headers */,
 				399C01E620BC17A10056D7AC /* MoppLibCryptoActions.h in Headers */,
-				C54EA6F62049641E0039AC78 /* ReaderInterface.h in Headers */,
```

Comment:
is this necessary?

Reply:
It seems to be change made by Xcode. Checked Build Phases -> Headers, the file exists there.